sormas-backend/src/main/java/de/symeda/sormas/backend/util/DtoHelper.java

```diff
@@ -70,7 +70,11 @@ public static void fillDto(EntityDto dto, AbstractDomainObject entity) {
 
 				Object targetValue = pd.getReadMethod().invoke(target);
 				Object sourceValue = pd.getReadMethod().invoke(source);
-				
+
+				if (sourceValue == null) {
```

Comment:
This is really, REALLY bad! It essentially makes it impossible to override target values with null in case they have been removed from the source, which is perfectly possible and viable. This change would result in really problematic bugs.

Reply:
was added as a fix for some errors i didn't understand at the moment and i forgot it there
will revert